### ISSUE TITLE
[A11Y][Input Text] Set accessibility hint on placeholder if no helper text

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1083]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone14+

### OS Version

iOS26

### App version

21 beta

### App build number

1761081723

### App build type

beta

### Describe the issue

The helper text allows giving some hint about text input.

According to component design, the helper text is not mandatory. 
<img width="383" height="179" alt="" src="https://github.com/user-attachments/assets/178e731f-f65b-430f-b69a-dbecfcc88445" />

If a visual hint is given in the placeholder, it will not be read by Voice Over

### Expected Behavior

If not helper-text, so the `.accessibilityHint ` should be set on the placeholder as a fallback

### Accessibility settings

Voice Over

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1766009311**
- Build tag: **117a933**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
